platform/podjs/src/browserPod.ts

```diff
@@ -21,12 +21,26 @@ const NAV_DEFAULT_BACKGROUND_COLOR = "#ffffff";
 const NAV_DARK_FOREGROUND_COLOR = "#000000";
 const NAV_LIGHT_FOREGROUND_COLOR = "#ffffff";
 
+/**
+ * @class LocalStoragePolyIn implements the PolyIn interface by using local storage.
+ */
 class LocalStoragePolyIn implements PolyIn {
     private static readonly storageKey = "polyInStore";
+
+    /* Checking if there is a value in local storage for the key "polyInStore"
+    and if there is, it is parsing the value into an array. 
+    If there is no value, it is setting the value to an empty array. 
+    */
     private store = JSON.parse(
         localStorage.getItem(LocalStoragePolyIn.storageKey) || "[]"
     );
 
+    /**
+     * It takes a matcher object and returns a promise that resolves to an array
+     * of quads that match the matcher
+     * @param matcher - Partial<Matcher>
+     * @returns An array of quads.
+     */
```

Comment:
There's a lot of JSDoc (or TSDoc I guess?) added for methods we're just implementing from an interface, in this case `PolyIn`. Does it make sense to add redundant documentation for those methods? The behaviour described on the interface is exactly the behaviour this implementation should elicit, right? So I'd only expect JSDoc/TSDoc on the class itself, not on methods defined and documented in the interface.

Reply:
Well, taking into account that right now documentation for different modules is not linked, it's not *really* redundant.

Reply:
It's redundant regardless of whether anything is linked anywhere - we have two (different!) descriptions of what `polyIn.match` does in two different places. Even if it _was_ in sync, it would be impossible to keep it in sync. If it doesn't show up in the generated JSDoc for whatever reason, I'm sure we can use `@link` or however that works, but writing the same documentation twice in two places (or even just manually copying it) is not an option.